### DISCUSSION TITLE
Fast, Fast, Fast

### DISCUSSION THREAD

**Discussion by hatamiarash7** [projectdiscovery/httpx#1748] [Q&A]
I have a list of 200K domains. Is there any way to run `httpx` faster? The current speed is about 23K domains per hour... It's too low.

How can I increase this?

**OS**: Debian 11
**CPU**: E5-2697 v2 - 2.70GHz - 24 Core / 48 Thread

**Comment by Spidle** [comment]
Did you solve this? What command are you running?

**Reply by hatamiarash7** [reply]
Not really. The speed is not optimal for many records, and I couldn't solve this problem.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
